docs/control-structures.rst

```diff
@@ -118,14 +136,21 @@ External Function Calls
 The expressions ``this.g(8);`` and ``c.g(2);`` (where ``c`` is a contract
 instance) are also valid function calls, but this time, the function
 will be called "externally", via a message call and not directly via jumps.
-Please note that function calls on ``this`` cannot be used in the constructor, as the
-actual contract has not been created yet.
+
+..warning::
+  While it's possible to use ``address(this)`` to retrieve the address of the
```

Comment:
Please create a new PR just with this. Also `address(this)` is a complication, because this already applies to `this` - "possible to use ``this`` to retrieve a reference to the contract in the constructor, as ...`